float64 interval_time
int32 x_pos
int32 y_pos
int16 angule
uint8 us_1
uint8 us_2
uint8 us_3
uint8 us_4
uint8 platform_state
int16 linear_velocity
int16 angular_velocity


